engine/src/flutter/impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -505,9 +599,18 @@ fml::StatusOr<RenderTarget> MakeBlurSubpass(
                 linear_sampler_descriptor));
         GaussianBlurVertexShader::BindFrameInfo(
             pass, data_host_buffer.EmplaceUniform(frame_info));
+        KernelSamples full_kernel_samples = GenerateBlurInfo(blur_info);
+        // For bounded blurs, kernel samples must not be lerped, as this can
+        // introduce wave-like artifacts at the edges of the bounds. This is
+        // likely due to increased sensitivity to pixel accuracy caused by
+        // opacity normalization.
```

Comment:
I think we should try to address this.  Without the lerp hack the blurs are much more costly.  4x more costly since it doubles in both dimensions?  A lot of work has gone into making our blurs fast and apps are very sensitive to it.  What have you tried so far?

Reply:
This has been addressed. The bounded blur now uses the kernel hack perfectly.